engineering/onboarding-fresher.md

```diff
@@ -55,4 +55,21 @@ Problems:
 - None
 ```
 
-### GOAL 2: Complete [JTC Experienced Hire Onboarding](https://github.com/jalantechnologies/handbook/blob/main/engineering/onboarding-experienced-hire.md)
+### Demo Day:
+
+Every two weeks, there will be a demo day where you will be asked to present all the projects you completed as part of curriculum.
```

Comment:
Nit: It's not clear what's the objective of this demo would be. Would we evaluate the candidate based on the progress? Or are we just checking in to see what's the progress that has been made so far?

Reply:
Both.

Updated the section with objective.